external-crates/move/crates/move-vm-runtime/src/execution/dispatch_tables.rs

```diff
@@ -183,183 +149,46 @@ impl VMDispatchTables {
         &self,
         vtable_key: &VirtualTableKey,
     ) -> PartialVMResult<VMPointer<Function>> {
-        let Some(result) = self
+        if let Some(result) = self
```

Comment:
nit: may be a good usage of `map_err` here, but I could see this being fine either way.

Reply:
I want the explicit `ptr_clone()` in the code -- it _would_ automatically copy, but it seems prudent to spell out what is happening.